.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--- Please provide a general summary of your changes in the Title above -->
+
+# PR Description
+
+<!--- Describe the changes your PR makes in detail -->
+
+References issue: # (ISSUE)
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Tests and Coverage
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how your change affects other areas of the code, etc. -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Docs (Added to or improved Databay's documentation)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Final Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+
+- [ ] My code follows the code style of this project.
```

Comment:
Do we say what the code style is anywhere? If not we should write it somewhere. And in either case we could link to that place here.

Reply:
'Somewhere' == probably in the CONTRIBUTING

Reply:
We don't so yeah would be good actually to add that into the CONTRIBUTING page.

Reply:
Cool, are you going to add that into this PR or would you see it as a separate contribution?

Reply:
What I'll do is make the words`code style` a link to the CONTRIBUTING page where we can add the code style in. It should hopefully encourage them to read that too.

Reply:
Cool, good idea. Are you thinking to make the actual change to the CONTRIBUTING tho or leave that for a separate PR?

Reply:
Hey @cdpierse so I think this thread is still pending on this PR?